src/views/webviews/welcome/welcome.ts

```diff
@@ -0,0 +1,165 @@
+              <p>●&nbsp;&nbsp;&nbsp;Effortlessly communicate across GitHub, Slack, and VS Code.</p>
+              <p>●&nbsp;&nbsp;&nbsp;Engage in meaningful conversations with contextual AI during PR discussions.</p>
+              <p>●&nbsp;&nbsp;&nbsp;Stay fully engaged in your ongoing code reviews.</p>
+              <p>●&nbsp;&nbsp;&nbsp;Stay updated in real-time on CI/CD and Automation.</p>
+              <p>●&nbsp;&nbsp;&nbsp;Execute actions on your code reviews using natural language.</p>
+            </ul>
+          </div>
+          <div>
+
+            <h3>My Active Pull Requests</h3>
+            <p><strong>PR Dashboard:</strong> Our extension equips VS Code with a PR Dashboard, offering a consolidated view
+              of your active pull requests. It enables quick identification and focus on pull requests needing attention,
+              all within VS Code Quick Pick.</p>
+            <img src="${assetsPath}/pr-dashboard.png" id="pr-dashboard" alt="PR-Dashboard" />
+          </div>
+          <div>
+            <h3>Quick Actions</h3>
+            <p>The extension comes with a set of actions designed to help you take care of your code review tasks and return
+              to your flow state. Actions you can take on pull request from VS Code are: </p>
+            <ul>
+              <li>&nbsp;&nbspOpen in Slack</li>
+              <li>&nbsp;&nbspOpen in GitHub</li>
+              <li>&nbsp;&nbspSend message to author/reviewer on Slack</li>
+              <li>&nbsp;&nbspApprove pull request</li>
+              <li>&nbsp;&nbspAdd labels on pull requests</li>
+              <li>&nbsp;&nbspAdd reviewers on pull requests</li>
+              <li>&nbsp;&nbspAdd assignees on pull requests</li>
+            </ul>
+            <div>
+              <img src="${assetsPath}/pr-chat.gif" id="pr-dashboard" alt="PR-Chat-Image" />
+            </div>
+          </div>
+        </section>
+        <section class=column id="second-column">
+          <div id="get-started-section">
+            <h3>Get Started</h3>
+            <div>
+              <span> Pullflow is integrated with GitHub, Slack, and VS Code to help you merge PRs 4X faster. Here is
+                how you can get started.</span>
+            </div>
+            <div>
+              <ol>
+                <li>Sign up on <a href="https://app.pullflow.com"> Pullflow</a> and connect your Slack and GitHub accounts.
+                </li>
+                <li> Click on <strong>Sign in to Pullflow</strong> from the VS Code status bar. </li>
+                <li> Thats it! See statuses of your pull requests in status bar. </li>
+                <ol>
+            </div>
+          </div>
+          <div id="command-div">
+            <h3>Commands</h3>
+            <table>
+              <tbody>
+                <tr>
+                  <td>Pullflow: Active Pull Requests (Ctrl/Cmd+Shift+,)</td>
+                </tr>
+                <tr>
+                  <td>Pullflow: Login</td>
```

Comment:
We use 'sign in' (not login) in most places. Let's make it consistently 'Sign in'.

Reply:
@hassan-pullflow this is still pending